README.md

```diff
@@ -1,92 +1,73 @@
 # Fluence JS Client
 
-[![npm](https://img.shields.io/npm/v/@fluencelabs/js-client.api?label=@fluencelabs/js-client.api)](https://www.npmjs.com/package/@fluencelabs/js-client.api)
-[![npm](https://img.shields.io/npm/v/@fluencelabs/js-client.web.standalone?label=@fluencelabs/js-client.web.standalone)](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone)
+[![npm](https://img.shields.io/npm/v/@fluencelabs/js-client?label=@fluencelabs/js-client)](https://www.npmjs.com/package/@fluencelabs/js-client)
 
 This is the Javascript client for the [Fluence](https://fluence.network) network. The main role of the JS client is to connect to the Fluence Network and allow you to integrate Aqua code into your application.
 
 ## Installation
 
-Adding the Fluence JS client for your web application is very easy.
+> JS Client only supports the ESM format that means not every Node.js project can install it.
+> You can read more [here](https://nodejs.org/api/esm.html)
 
-### Browser-based Apps
+1. Install the client:
 
-1. Add a script tag with the JS Client bundle to your `index.html`. The easiest way to do this is using a CDN (like [JSDELIVR](https://www.jsdelivr.com/) or [UNPKG](https://unpkg.com/)). The script is large, thus we highly recommend to use the `async` attribute.
-
-   Here is an example using the JSDELIVR CDN:
-
-   ```html
-   <head>
-     <title>Cool App</title>
-     <script
-       src="https://cdn.jsdelivr.net/npm/@fluencelabs/js-client.web.standalone@0.13.3/dist/js-client.min.js"
-       async
-     ></script>
-   </head>
+   ```bash
+   npm i @fluencelabs/js-client
    ```
 
-   If you cannot or don't want to use a CDN, feel free to get the script directly from the [npm package](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone) and host it yourself. You can find the script in the `/dist` directory of the package. (Note: this option means that developers understand what they are doing and know how to serve this file from their own web server.)
-
-2. Install the following packages:
-
-   ```
-   npm i @fluencelabs/js-client.api @fluencelabs/fluence-network-environment
-   ```
-
-3. Add the following lines at the beginning of your code:
+2. Add the following lines at the beginning of your code:
 
-   ```
-   import { Fluence } from "@fluencelabs/js-client.api";
-   import { randomKras } from '@fluencelabs/fluence-network-environment';
+   ```javascript
+   import { Fluence, randomKras } from "@fluencelabs/js-client";
 
    Fluence.connect(randomKras());
```

Comment:
does this work already again? Did you fix this issue that required users to pass config like this `Fluence.connect(randomKras(), {});`

Reply:
Yes, i changes API in this PR